core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamlet.scala

```diff
@@ -36,7 +36,7 @@ abstract class AkkaStreamlet extends Streamlet {
   final override val runtime = AkkaStreamletRuntime
 
   // ctx is always first set by runner through `init` so this is safe.
-  private var ctx: AkkaStreamletContext = null
+  @volatile private var ctx: AkkaStreamletContext = null
```

Comment:
@RayRoestenburg What about Spark and Flink streamlets ? Should we not make the context `volatile` there ?

Reply:
Good point!